lib/steep/drivers/utils/driver_helper.rb

```diff
@@ -4,13 +4,23 @@ module Utils
       module DriverHelper
         attr_accessor :steepfile
 
-        def load_config(path: steepfile || Pathname("Steepfile"))
-          raise "Cannot find a configuration at #{path}: `steep init` to scaffold" unless path.file?
-
-          steep_file_path = path.absolute? ? path : Pathname.pwd + path
-          Project.new(steepfile_path: steep_file_path).tap do |project|
-            Project::DSL.parse(project, path.read, filename: path.to_s)
-
+        def load_config(path: steepfile || Pathname("Steepfile"), warnings: true)
+          if path.file?
+            steep_file_path = path.absolute? ? path : Pathname.pwd + path
+            Project.new(steepfile_path: steep_file_path).tap do |project|
+              Project::DSL.parse(project, path.read, filename: path.to_s)
+            end
+          else
+            if warnings # Silence warnings in children worker processes
+              warn "Cannot find a configuration at #{path}: `steep init` to scaffold. Using current directory..."
```

Comment:
`Steep.logger.error` ?

Reply:
Never realized Steep uses a logger system. I will check it out.

Reply:
I was afraid if `warn` is hidden by default, but confirmed the default log level is `info`.
So, `Steep.logger.warn` would be the best one.

Reply:
I’ve replaced it with `Steep.logger.warn` and written tests but haven’t pushed the changes, for I discovered that `Steep.logger.warn` doesn’t print whereas both `Steep.logger.error` and plain `warn` are fine. Let me know if this is intended and which direction I should move forward.

Reply:
@ParadoxV5 I'm sorry. It's my bad. `ERROR` is the default value and it should be `Steep.logger.error`. 🙇‍♂️ 

https://github.com/soutaro/steep/blob/master/lib/steep.rb#L158